sensors/external_pressure_sensor:
  depth_offset: -0.36411

